.github/workflows/github-page.yml

```
@@ -14,54 +15,47 @@ jobs:
         shell: bash
 
     steps:
-    - name: Install Nix
-      uses: cachix/install-nix-action@v18
-      with:
-        # Use last stable nixos channel and the same nix as in channel:
-        install_url: https://releases.nixos.org/nix/nix-2.11.1/install
-        nix_path: nixpkgs=channel:nixos-22.11
-        github_access_token: ${{ secrets.GITHUB_TOKEN }}
-        extra_nix_config: |
-          access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
-          experimental-features = nix-command flakes
-          allow-import-from-derivation = true
-          substituters = https://cache.nixos.org https://cache.iog.io
-          trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+      - uses: actions/checkout@v4
 
-    - uses: actions/checkout@v4
+      - uses: cachix/install-nix-action@v30
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
+          extra_nix_config: |
+            trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
+            substituters = https://cache.iog.io/ https://cache.nixos.org/
 
-    - name: Fetch nix cache and update cabal indices
-      run: |
-        nix develop .\#haddockShell --command \
+      - name: Activate default nix shell
+        uses: rrbutani/use-nix-shell-action@v1
```

```suggestion
      - uses: rrbutani/use-nix-shell-action@v1
```

Rationale: be consistent with other files